src/main/java/echo/Parser.java

```diff
@@ -0,0 +1,111 @@
+package echo;
+
+
+/**
+ * The Parser class is responsible for interpreting user commands
+ * and performing the corresponding actions on the task list.
+ */
+public class Parser {
+
+    public static String listAllTask(TaskList allTasks) {
```

Comment:
Method name 'listAllTasks' might be better + this would make it more consistent with the 'allTasks' variable as well